src/Controller/TalkCommentsController.php

```diff
@@ -189,4 +194,13 @@ public function updateComment(Request $request, PDO $db)
 
         return $commentMapper->getCommentById($commentId);
     }
+
+    private function getCommentMapper(Request $request, PDO $db): TalkCommentMapper
+    {
+        if ($this->commentMapper === null) {
```

Comment:
Unless we assign `$this->commentMapper` a value, this condition will always evaluate to `true`.